USERS.md

```diff
@@ -0,0 +1,28 @@
+# Who is using Karpenter?
+Karpenter has a variety of users and use cases for scaling Kubernetes.
+Many customers want to learn from others who have already implemented Karpenter in their environments.
+
+The following is a self-reported list of users to help identify adoption and points of contact.
+
+## Community
+If you would like to ask question from the community please join the [Karpenter slack channel in the Kubernetes Slack](https://kubernetes.slack.com/archives/C02SFFZSA2K) or join the [Karpenter working group](WORKING_GROUP.md) and fort-nightly calls.
+
+## Add yourself as a user
+If you are using Karpenter please consider adding yourself as a user by opening a pull request to this file.
+If you are open to others contacting you about your use of Karpenter on Slack, add your Slack name as well.
+
+### Example entry
+
+* *Amazon, Inc.*
+* Scaling production workloads and batch jobs in 4 regions
+* [Introducing Karpenter](https://aws.amazon.com/blogs/aws/introducing-karpenter-an-open-source-high-performance-kubernetes-cluster-autoscaler/)
+* `@Alex Kestner`, `@Ellis Tarn`
+
+```
+* *Company*
+* Description of where Karpenter is used
+* _Link to company or blog/details_ (optional)
+* `slack user`, `email@company` (optional)
+```
+
+## Users (Alphabetically)
```

Comment:
Hard to see in the preview if this works, but you can try this.
```suggestion
## Adopters

| Organization                      | Product/Project Name                                                   | Status                   | More Information                                                     |                K8s Slack Contact
| --------------------------------- | ---------------------------------------------------------------------- | ------------------------ | -------------------------------------------------------------------- |
| Amazon, Inc.                      | Use-Case                                               | Production               | [link](https://cs.console.aliyun.com/)                               |            `@Alex Kestner` and `@Ellis Tarn`
```

Reply:
This redirects me to alibaba: https://cs.console.aliyun.com/

Reply:
oops!